### PR TITLE
To-do-list _with-react

### PR DESCRIPTION
**Pull request by Said-laasri**
React Project: To-do-List
In this PR, the following were implemented:-

- No linter errors
- Use the correct Gitflow
- Implemented Add todo function
- Implemented Edit todo function
- Implemented Update todo function

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,83 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+- Inspiration
+- etc
```
Comment (amejid) [inline README.md:83]: It would be nice to mention the original author of the work,
you can mention his [Github](https://github.com/Ibaslogic)

**Review by amejid — APPROVED** [review]
## STATUS: APPROVED 🥇 🥇 🥇 

Hi @Said-laasri , Good job


- No linter errors
- Followed the correct Gitflow
- Features working as expected 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 💯

**Optional suggestions**

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

 Please Check the comments under the review.
Cheers and Happy coding! 👏
<hr>
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
<hr>

File: README.md (hunk 1 above)
Comment (amejid) [inline README.md]: It would be nice to mention the original author of the work,
you can mention his [Github](https://github.com/Ibaslogic)

Comment (Said-laasri) [reply README.md]: @amejid thanks for the review 
<hr>
change is done

**Comment by Said-laasri** [comment]
@amejid  Thank you for reviewing my project

**Pull request merged**